app/docker/views/containers/create/createcontainer.html

```diff
@@ -281,7 +290,7 @@
                         <div class="btn-group btn-group-sm">
                           <label class="btn btn-primary" ng-model="volume.readOnly" uib-btn-radio="false">Writable</label>
                           <label class="btn btn-primary" ng-model="volume.readOnly" uib-btn-radio="true">Read-only</label>
```

Comment:
In my opinion, instead of using a true/false value here, we should support Writable, Read-only, z & Z. I believe that this is better than a switch as all values are mutually exclusive (e.g. cannot create a volume with ro & z).

Although we might want to figure out a better description for these button that z & Z, something more user friendly.